_posts/2021-05-28-seqan-3-0-3-release.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "SeqAn 3.0.3 released"
+categories: release
+excerpt_separator: <!--more-->
+redirect_from:
+  - /seqan-3-0-3-release/
```

Comment:
```suggestion
```

not needed any more as this is a new release and was not available on our old website